templates/includes/transactions.tx

```diff
@@ -11,9 +11,11 @@
         <div class="one-filing rejected">
         % if $transaction.company_name {
             <h2 class="heading-medium">
-                <a id="company-name-<% $~transaction.count %>" href="/company/<% $transaction.company_number %>"><% $transaction.company_name %></a>
                 % if $transaction.company_number {
```

Comment:
There's a 'transaction.tx' template file in the same directory which has similar mark-up logic but I can't see where it's used and if it also requires changing. Thanks @caismith-ch for checking this and helping.